spec.html

```diff
@@ -22038,7 +22032,7 @@ <h1>
             1. Set the running execution context's LexicalEnvironment to _newEnv_.
           1. Let _exprRef_ be Completion(Evaluation of _expr_).
           1. Set the running execution context's LexicalEnvironment to _oldEnv_.
-          1. Let _exprValue_ be ? GetValue(_exprRef_).
+          1. Let _exprValue_ be ? GetValue(? _exprRef_).
```

Comment:
Out of curiosity did ecmarkup find this or was this manually audited?

Reply:
Manually, alas. ecmarkup does not yet track types except for checking the completion-ness of callsites of AOs, which is purely local.